### ISSUE TITLE
unable to initialize spinnaker? 

### ISSUE THREAD

**Issue by nsteinme** [iorodeo/bias#2]
Hi, this software was recommended to me by some colleagues at Janelia, hoping you can help figure out my error? 

I get the error message below ("failed to connect" / 102 / "unable to initialize spinnaker" / -1006) when clicking "connect". 

I'm using release 0.62 (the pre-built executable) on windows. If I run SpinView, the cameras (there are two) both come up fine and display live images as expected. When I run BIAS it does open two windows - corresponding I assume to the two cameras - so that seems promising. I tried running the "_debugwin" version but there's nothing displayed in the command prompt. I couldn't find anything about these error codes or error text on some google searching. Any suggestions/help much appreciated, thanks. 

![image](https://user-images.githubusercontent.com/7590975/100040416-14bd8b80-2dbc-11eb-926e-595493385dc5.png)

**Comment by willdickson** [comment]
Hi Nick,  it looks like the error you are getting (-1006)  is SPINNAKER_ERR_INVALID_HANDLE. This is happening when  trying tp initialize the camera with spinCameraInit(hCamera_);     I'm not sure why the handle is invalid.  What camera are you using?

**Comment by nsteinme** [comment]
Thanks for getting back to me - each camera is a PointGrey/FLIR
Chameleon CM3-U3-13Y3M-CS. I guess they are fundamentally compatible with
spinnaker given that they open fine in spinview.

**Comment by willdickson** [comment]
Yes, it definitely should be compatible.   Spinview is closed when you try the camera with BIAS?

**Comment by nsteinme** [comment]
Yes, and I tried restarting the computer to try clearing any "holds" or
un-closed processes, and immediately trying BIAS without running spinview
first - same thing.

**Comment by LiuDaveLiu** [comment]
Hi, has this been resolved? I have the same error.

Thank you,

Dave

**Comment by nsteinme** [comment]
I didn't get it working - just gave up. :(
